rule.go

```diff
@@ -412,6 +419,21 @@ func (p PCRE) String() string {
 	return s.String()
 }
 
+//string returns a string for all of the flowbits
+func (fb Flowbit) String() string {
```

Comment:
You'll need to add tests for this (see other String() tests)[0]

[0] https://github.com/google/gonids/blob/master/rule_test.go#L502